python-sdk/src/astro/databases/base.py

```diff
@@ -415,7 +415,7 @@ def load_file_to_table(
         native_support_kwargs: dict | None = None,
         columns_names_capitalization: ColumnCapitalization = "original",
         enable_native_fallback: bool | None = LOAD_FILE_ENABLE_NATIVE_FALLBACK,
-        load_options: LoadOptions | None = None,
+        load_options: LoadOptions = LoadOptions(),
```

Comment:
I'm thrilled with how we're heading into unifying the `LoadOptions` and having subclasses specific to the database. It is a significant improvement since we introduced `native_kwargs`.

I was curious about the design decision here to create, by default, an instance of the class - instead of having `None`. Could you share the reason, @dimberman?

Reply:
@tatiana I was worried we'd have to create Loadoptions the whole way down as certain functions require one but I found a workaround so was able to get rid of this.